protein/model_func.py

```diff
@@ -0,0 +1,45 @@
+from protein.models import ProteinFamily
+
+""" This is a solution for new builds with change of the definition of CLASSLESS_PARENT_GPCR_SLUGS
```

Comment:
CODACY is complaining about """. We should set up CODACY to ignore this specific issue or turn off the Pylint code pattern "pointless-string-statement (W0105)" from the policy coding standard.

Reply:
No need to change CODACY setup. The """ must be inside a function or a class at the beginning. That way Codacy seems to know it is documentation. But, this is not practical for documenting modules at the beginning of the file.

I just put the comment inside the function as right now it is only a single function.
Solved in [bd93eac](https://github.com/protwis/protwis/pull/550/commits/bd93eacbf732fe5f39371d82ff0c5b6d4e693bcf).